docs/index.bs

```diff
@@ -2932,6 +2940,7 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
       1. Let |client| be |request|'s [=request/client=].
       1. Let |reservedClient| be |request|'s [=request/reserved client=].
       1. Let |preloadResponse| be a new [=promise=].
+      1. Let |eventHandled| be a new [=promise=].
```

Comment:
We're a little inconsistent here because specs were still being figured out when we first wrote this, but the above should use [=a new promise=], which points to https://heycam.github.io/webidl/#a-new-promise.

You should also specify a realm for this, so this line should come just before line 20, when you have the active worker, and therefore "the [=relevant realm=] of the |activeWorker|'s [=service worker/global object=]".

Reply:
Thanks. I changed to declare |eventHandled| to be null fist in top-level to make it able to be accessed in those last steps. Then set it to a new promise when |activeWorker| is available. Hope this could also cover your idea.